src/index.js

```diff
@@ -258,12 +285,24 @@ router.get('/artists', async (ctx, next) => {
       .sort({
         last_activity: -1
       })
-      .select('-_id name twitter_handle label bio city country')
-      .skip(perPage * page)
-      .limit(perPage)
+      .select('-_id user_id name twitter_handle label bio city country')
+      .skip(limit * page)
+      .limit(limit)
 
     ctx.body = {
-      data: result
+      data: await map(result, async (item) => {
```

Comment:
I might be misremembering, but `await Promise.all(result.map(async item => {})` Would work right, without needing an outside library?

Reply:
Yes, of course, it's just i've been using this lib before and it's useful for chaining and writing cleaner code.